test/artefacts/simplified-expenses-checker/car/live_on_business_premises/no/no/new/2345.0/no/250001.0/35.0/10000/789.0/0.txt

```diff
@@ -0,0 +1,40 @@
+
+
+Based on your answers, the expenses you can claim and deduct are listed below. There are 2 sets of results comparing simplified expenses and calculating expenses by working out the actual costs.
+
+##Simplified expenses
+
+- £4,500 to claim for cars and vans
+
+**Total to claim per year:** £4,500
+
+###What to do for your tax return
+
+Use the simplified expenses flat rates to work out your business expenses.
+
+Multiply the flat rates with the [miles you drove](/simpler-income-tax-simplified-expenses/vehicles-), [hours you work from home](/simpler-income-tax-simplified-expenses/business-use-of-your-home) or the [number of people who live on your business premises](/simpler-income-tax-simplified-expenses/private-use-of-business-premises).
+
+[Self Assessment tax returns](/self-assessment-tax-returns)
+
+##Calculating expenses by working out the actual costs
+
+- £2,345 for cars and vans and motorcycles
+
+- £15,750.06 for for cars, vans and motorcycles
```

Comment:
Is the content being fixed? I can see that the content for `motorbikes` is being fixed in commit 7d207bd, but what about `cars`?